training/templates/sklearn/template/README.md

```diff
@@ -0,0 +1,186 @@
+# Scikit-learn trainer template for AI Platform
+
+
+This is a template for building a scikit-learn-based machine learning trainer 
+that can be run on AI Platform. 
+
+Google Cloud tools used:
+- [Google Cloud Platform](https://cloud.google.com/) (GCP) lets you build and 
+host applications and websites, store data, and analyze data on Google's 
+scalable infrastructure.
+- [Cloud ML Engine](https://cloud.google.com/ml-engine/) is a managed service 
+that enables you to easily build machine learning models that work on any type 
+of data, of any size. This is now part of 
+[AI Platform](https://cloud.google.com/ai-platform/).
+- [Google Cloud Storage](https://cloud.google.com/storage/) (GCS) is a unified 
+object storage for developers and enterprises, from live data serving to data 
+analytics/ML to data archiving.
+- [Cloud SDK](https://cloud.google.com/sdk/) is a set of tools for Google Cloud 
+Platform, which contains e.g. gcloud, gsutil, and bq command-line tools to 
+interact with Google Cloud products and services.
+- [Google BigQuery](https://cloud.google.com/bigquery/) A fast, highly scalable, 
+cost-effective, and fully managed cloud data warehouse for analytics, with even 
+built-in machine learning.
+
+## Template structure
+```
+template 
+    |__ config
+        |__ config.yaml             # for running normal training job on AI Platform
+        |__ hptuning_config.yaml    # for running hyperparameter tunning job on AI Platform    
+    |__ scripts
```

Comment:
Our samples (due to modularization) are no longer following this template. We have separated training from prediction. (e.g. we have train-local.sh and train-cloud.sh, as well as many other differences).

Reply:
Renamed